rasa/core/policies/_ted_policy.py

```diff
@@ -0,0 +1,1094 @@
+# WARNING: This module will be dropped before Rasa Open Source 3.0 is released.
+#          Please don't do any changes in this module and rather adapt `TEDPolicy2` from
+#          the regular `rasa.core.policies.ted_policy` module. This module is a
+#          workaround to defer breaking changes due to the architecture revamp in 3.0.
+# flake8: noqa
```

Comment:
💡 Our time is too valuable to fix docstring errors in modules which we will drop anyway